### PR TITLE
Added Base Converter command

### PR DESCRIPTION
**Pull request by thesulaj**
## Description

Added the "bcv" (`B`ase `C`on`v`erter) command. Allows the user to easily change the base of one or a series of space-delimited numbers. Very useful for embedded developers, for example.

## Type of change

- [x] New script command

## Screenshot

Untitled

## Dependencies / Requirements

N/A

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/developer-utils/bcv.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title bcv
+# @raycast.mode compact
+# @raycast.refreshTime 1h
+
+# Optional parameters:
+# @raycast.icon 🤖
+# @raycast.packageName Base converter
```
Comment (unnamedd) [inline commands/developer-utils/bcv.sh:11]:
```suggestion
# @raycast.packageName Developer Utilities
```

File: commands/developer-utils/bcv.sh [hunk 2]
```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title bcv
+# @raycast.mode compact
+# @raycast.refreshTime 1h
```
Comment (unnamedd) [inline commands/developer-utils/bcv.sh:7]: This won't have any effect in this Script Command. `@raycast.refreshTime` only works together with the `@raycast.mode` `inline`.

```suggestion
```

File: commands/developer-utils/bcv.sh [hunk 3]
```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```
Comment (unnamedd) [inline commands/developer-utils/bcv.sh:1]: Please, rename the file to `base-converter.sh`.

File: commands/developer-utils/bcv.sh [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+# @raycast.packageName Base converter
+# @raycast.author Saimir S.
+# @raycast.authorURL saimirsulaj.com
+# @raycast.description Utility to convert numbers between bases. Can be one number or a list of string-delimited numbers.
+# @raycast.argument1 { "type": "text", "placeholder": "from base", "optional": false }
+# @raycast.argument2 { "type": "text", "placeholder": "to base", "optional": false }
+# @raycast.argument3 { "type": "text", "placeholder": "value", "optional": false }
+
+# Documentation:
+# @raycast.author Saimir S.
```
Comment (unnamedd) [inline commands/developer-utils/bcv.sh:20]: You already added it above, please remove it from here.
```suggestion
```

**Review by unnamedd — CHANGES_REQUESTED** [review]
Hi @thesulaj, I made some comments, when you finish to address them, please let us know for a second round of review.

Thanks for your first contribution and welcome! :D

**Comment by thesulaj** [comment]
Hi @unnamedd, thanks for the reviewing the PR. I addressed all of your comments, let me know if any further changes are required.

File: commands/developer-utils/base-converter.sh [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+for num in $3; do
+  if ! [[ $num =~ $num_re ]] ; then
+     echo "Error: \"$num\" is not a number" >&2;
+     exit 1;
+  fi
+  num_cov=$(echo "ibase=$input_base; obase=$output_base_cov; $num" | bc);
+  output="$output $num_cov";
+done
+
+echo "$output";
```
Comment (grzegorzkrukowski) [inline commands/developer-utils/base-converter.sh:59]: Hey

It seems that $output contains some trailing whitespace - may be a good idea to use `${output##}` before copying it to pasteboard.

Comment (thesulaj) [reply commands/developer-utils/base-converter.sh:62]: Great idea, implemented. Thanks for the feedback

**Pull request merged**